packages/cli/src/commands/Interfaces.ts

```diff
@@ -1,4 +1,6 @@
-interface IResult {
```

Comment:
we should move this file out of this folder, as `oclif` is trying to find a command in this file, and creates unnecessary noise in the logs.